prisma/migrations/20210710225536_added_reviews/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- CreateTable
+CREATE TABLE "Review" (
+    "id" SERIAL NOT NULL,
+    "movie_id" TEXT NOT NULL,
+    "author_id" INTEGER NOT NULL,
+    "content" TEXT,
+    "rating" DOUBLE PRECISION NOT NULL,
```

Comment:
Looks like I can get an accurate representation for 5/7 with this!

Reply:
lol. I was thinking there will be some validation before adding a row to restrict values to a range of 0-5 with increments of 0.5 but seems like at the database level this is enough?

Reply:
Yep!